pkg/store/cache/caching_bucket.go

```diff
@@ -29,12 +29,6 @@ import (
 const (
 	originCache  = "cache"
 	originBucket = "bucket"
-
-	opGet        = "get"
-	opGetRange   = "getrange"
```

Comment:
This was actually mismatching what we have already for bucket.

Reply:
Doh! I think we should mention this change in the CHANGELOG

Reply:
Any mixin / alert we could change?

Reply:
Nothing, I could find.